src/app/logic/PlayerHuman.js

```diff
@@ -0,0 +1,13 @@
+class PlayerHuman {
```

Comment:
Note for next task: Perhaps we can create a class Player with methods, and then extend it by creating HumanPlayer and ComputerPlayer